src/Sylius/Bundle/AdminBundle/templates/product_option/form/sections/values/value/code.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="col-12 col-md-6">
```

Comment:
```suggestion
<div class="col-12">
```
🤔

Reply:
i need to have it to keep code and position next to each other